src/CharacterManager.cpp

```diff
@@ -90,7 +90,6 @@ Character* CharacterManager::GetDefaultCharacter()
 
 	/* We always have the default character. */
 	FAIL_M("There must be a default character available!");
-	return nullptr;
```

Comment:
Should I suppress the compiler warning in this case? The compiler complains because `FAIL_M()` will result in the application exiting, so we never actually reach the `return` line.

I made similar changes in a few other places.